server/controllers/list.js

```diff
@@ -1144,6 +1146,13 @@ const changeType = (req, resp) => {
           ? viewersIds.filter(id => isMember(list, id))
           : [...viewersIds, ...cohortMembers.filter(id => !isViewer(list, id))];
 
+      const viewersDiff = _difference(viewersIds, updatedViewersIds);
```

Comment:
remove this const and use the `_difference(viewersIds, updatedViewersIds)` in line 1153